types/stripe-js/elements/express-checkout.d.ts

```diff
@@ -1,184 +1,203 @@
-  ): StripePayButtonElement;
+    handler: (event: {elementType: 'expressCheckout'}) => any
+  ): StripeExpressCheckoutElement;
   once(
     eventType: 'cancel',
-    handler: (event: {elementType: 'payButton'}) => any
-  ): StripePayButtonElement;
+    handler: (event: {elementType: 'expressCheckout'}) => any
+  ): StripeExpressCheckoutElement;
   off(
     eventType: 'cancel',
-    handler?: (event: {elementType: 'payButton'}) => any
-  ): StripePayButtonElement;
+    handler?: (event: {elementType: 'expressCheckout'}) => any
+  ): StripeExpressCheckoutElement;
 
   /**
    * Triggered when a buyer selects a different shipping address.
    */
   on(
     eventType: 'shippingaddresschange',
-    handler: (event: StripePayButtonElementShippingAddressChangeEvent) => any
-  ): StripePayButtonElement;
+    handler: (
+      event: StripeExpressCheckoutElementShippingAddressChangeEvent
+    ) => any
+  ): StripeExpressCheckoutElement;
   once(
     eventType: 'shippingaddresschange',
-    handler: (event: StripePayButtonElementShippingAddressChangeEvent) => any
-  ): StripePayButtonElement;
+    handler: (
+      event: StripeExpressCheckoutElementShippingAddressChangeEvent
+    ) => any
+  ): StripeExpressCheckoutElement;
   off(
     eventType: 'shippingaddresschange',
-    handler?: (event: StripePayButtonElementShippingAddressChangeEvent) => any
-  ): StripePayButtonElement;
+    handler?: (
+      event: StripeExpressCheckoutElementShippingAddressChangeEvent
+    ) => any
+  ): StripeExpressCheckoutElement;
 
   /**
    * Triggered when a buyer selects a different shipping rate.
    */
   on(
     eventType: 'shippingratechange',
-    handler: (event: StripePayButtonElementShippingRateChangeEvent) => any
-  ): StripePayButtonElement;
+    handler: (event: StripeExpressCheckoutElementShippingRateChangeEvent) => any
+  ): StripeExpressCheckoutElement;
   once(
     eventType: 'shippingratechange',
-    handler: (event: StripePayButtonElementShippingRateChangeEvent) => any
-  ): StripePayButtonElement;
+    handler: (event: StripeExpressCheckoutElementShippingRateChangeEvent) => any
+  ): StripeExpressCheckoutElement;
   off(
     eventType: 'shippingratechange',
-    handler?: (event: StripePayButtonElementShippingRateChangeEvent) => any
-  ): StripePayButtonElement;
+    handler?: (
+      event: StripeExpressCheckoutElementShippingRateChangeEvent
+    ) => any
+  ): StripeExpressCheckoutElement;
 
   /**
-   * Updates the options the `PayButtonElement` was initialized with.
+   * Updates the options the `ExpressCheckoutElement` was initialized with.
    * Updates are merged into the existing configuration.
    */
-  update(options: StripePayButtonElementUpdateOptions): StripePayButtonElement;
+  update(
+    options: StripeExpressCheckoutElementUpdateOptions
+  ): StripeExpressCheckoutElement;
 };
 
-export type PaymentMethodType = 'google_pay' | 'apple_pay';
+export type PaymentMethodType = 'google_pay' | 'apple_pay' | 'link';
```

Comment:
I assume this is intentional with the Link in GA from a few days back?

Reply:
yep!